### ISSUE TITLE
Rank-consistent ordinal regression for neural networks

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#22]
https://arxiv.org/abs/1901.07884 
A loss function in the task of classifying rank data, such as age prediction, is proposed. Labels are defined as [age 10+], [age 20+], etc. and consistent predictions are theoretically guaranteed by using bias in each label.

![image](https://user-images.githubusercontent.com/24524018/89603793-c9cb4700-d8a4-11ea-9ad7-684830a9105e.png)
![image](https://user-images.githubusercontent.com/24524018/89603796-cc2da100-d8a4-11ea-9a48-2eeae05907b7.png)
